masternodes/setup-testnet.rst

```diff
@@ -1015,72 +1008,43 @@ Next, we will install the Dash Platform services. Start with some common
 dependencies::
 
   cd
-  curl -sL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
-  sudo apt install -y libzmq3-dev nodejs build-essential
-  sudo npm install forever -g
-
-MongoDB
-^^^^^^^
-
-MongoDB is a document-oriented database. Install MongoDB as follows::
-
-  sudo gpg --no-default-keyring --keyring /usr/share/keyrings/mongodb-archive-keyring.gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys 20691EEC35216C63CAF66CE1656408E390CFB1F5
-  echo "deb [signed-by=/usr/share/keyrings/mongodb-archive-keyring.gpg] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/4.4 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb.list
-  sudo apt update
-  sudo apt install -y mongodb-org
-
-Configure MongoDB::
-
-  sudo sed -i 's/#replication:/replication:\n  replSetName: driveDocuments/' /etc/mongod.conf
-
-Restart MongoDB::
-
-  sudo systemctl restart mongod
-
-Initiate the MongoDB client::
-  
-  mongo<<<"rs.initiate({_id:'driveDocuments',version: 1,members:[{_id: 0,host: 'localhost:27017',},],});"
-
-Verify MongoDB is running::
-
-  sudo systemctl status mongod
+  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
+  source ~/.bashrc
+  nvm install --lts
```

Comment:
Is platform committed to LTS support or should this be tied to a specific version? Looks like the next LTS bump is in less than 6 months (https://nodejs.org/en/about/releases/). I'd prefer using whichever option is less likely to result in broken docs in the future.